src/classes/HiFiMixerSession.ts

```diff
@@ -583,7 +594,11 @@ export class HiFiMixerSession {
      * Disconnects from the Mixer. Closes the RAVI Signaling Connection and the RAVI Session.
      * @returns A Promise that _always_ Resolves with a "success" status string.
      */
-    async disconnectFromHiFiMixer(): Promise<string> {
+    async disconnectFromHiFiMixer(internal:boolean = false): Promise<string> {
+        if (!internal) {
+            this._raviDiagnostics.noteExplicitApplicationClose();
+            this._hifiDiagnostics.noteExplicitApplicationClose();
+        }
```

Comment:
I'm not crazy about this. I want the report to distinguish between the application calling HiFiCommunicator.disconnectFromAudioAPIServer, vs our own internal calls to here. So I added a an internal:boolean argument and decorated our internal calls. Alternative suggestions welcome.

Reply:
yes, I like your suggestion better than mine.... and dislike them both.

Reply:
done